src/python/pants/backend/python/rules/pytest_coverage.py

```diff
@@ -141,18 +141,25 @@ class CoverageConfig:
 @rule
 async def create_coverage_config(coverage_config_request: CoverageConfigRequest) -> CoverageConfig:
     sources = await Get[SourceFiles](
-        AllSourceFilesRequest((tgt.get(Sources) for tgt in coverage_config_request.targets))
+        AllSourceFilesRequest(
+            (
+                tgt.get(PythonSources)
```

Comment:
Nit: reads better to use `tgt[PythonSources]`, as you already guaranteed the field exists via the `tgt.has_field`.

```suggestion
                tgt[PythonSources]
```